docs/development-setup.md

```diff
@@ -107,9 +107,9 @@ git checkout develop
 
 1. Install the required gems by running `bundle install` in the project directory.
 1. Install the required Node modules by running `yarn install` in the project directory.
-1. Copy `config/database.yml.example` to `config/database.yml` and edit `database.yml` to point to your database.
-1. Copy `config/settings.local.yml.example` to `config/settings.local.yml` and adjust settings as appropriate.
-1. Copy `config/initializers/local_config.rb.example` to `config/initializers/local_config.rb` and adjust settings as appropriate. Note that the reCAPTCHA and Google Maps API Key must be valid keys for those services in order for tests to pass.
+1. Run `cp config/database.yml.example config/database.yml` and edit `database.yml` to point to your database (typically *nemo_development*).
```

Comment:
Can we remove the second part of this one since we've standardized the name above?